Add ogre example application

compomega opened the pull request.

fincs commented:
This should be using romfs instead of an absolute folder on the root of the SD card.

compomega commented:
I haven't looked at romfs yes. Would that require a special file system abstraction to be added to the library?

fincs commented:
No, romfs works just fine with standard file access functions.

compomega commented:
I've updated this to complement the changes in the ogre library. I disabled the logging (but you can always enable it by changing those lines) as well as a shader cache that was writing to the SD card. I don't see anything being written into the SD card anymore. I've also changed it to use romfs (though it's nice to know it works with SD card access too).
